Travel Scheduler/Helper Files/APIManager.m

```diff
@@ -249,7 +249,7 @@ - (void)getDistanceFromOrigin:(NSString *)origin toDestination:(NSString *)desti
 //Departure time must be an integer in seconds since midnight, January 1, 1970 UTC
 - (void)getCommuteDetailsFromOrigin:(NSString *)originId toDestination:(NSString *)destinationId withDepartureTime:(int)departureTime withCompletion:(void (^)(NSArray *commuteDetailsArray, NSError *error))completion
```

Comment:
Maybe erase the departure time from the header since we are not using it anymore?